src/Sulu/Bundle/AdminBundle/Resources/js/services/Router/Router.js

```diff
@@ -150,6 +166,28 @@ export default class Router {
         return url + (queryString ? '?' + queryString : '');
     }
 
+    createAttributesHistory() {
+        if (!this.route) {
+            return;
+        }
+
+        if (!(this.route.name in this.attributesHistory)) {
+            this.attributesHistory[this.route.name] = [];
+        }
+
+        this.attributesHistory[this.route.name].push(toJS(this.attributes));
+    }
+
+    isRouteChanging(name: string, attributes: Object) {
+        const route = routeRegistry.get(name);
+
+        return !(
+            this.route
+            && this.route.name === route.name
+            && equal(this.attributes, attributes)
```

Comment:
The `MediaOverview` still makes problems... The thing is that the ID changes its datatype from number to string when navigating with a number, which causes two history entries to be written. We've already fixed that for observable by using `==` instead of `===`. But here we are currently using the `fast-deep-equal` package, which asserts by type. So we have two options here:

* Remove the `fast-deep-equal` package and write the function to compare the attributes on our own.
* Always try to parse a number with each attribute.

Reply:
We have decided to go for the second option, and always try to parse the attributes as numbers.